wpeverest/user-registration
# UR-930 Enhance - Add new addons to extensions page

### All Submissions:






### Changes proposed in this Pull Request:

The 4 new add-ons that were released in v3.0 are listed in the extensions JSON file.

### How to test the changes in this Pull Request:

Please make sure the add-ons are listed and displayed properly on the extensions page.

### Types of changes:




### Other information:




### Changelog entry

> Enhance - Add new addons to extensions page
